tikv-client/src/main/java/com/pingcap/tikv/txn/LockResolverClientV2.java

```diff
@@ -78,35 +76,8 @@ public LockResolverClientV2(
     readWriteLock = new ReentrantReadWriteLock();
   }
 
-  private void saveResolved(long txnID, long status) {
-    try {
-      readWriteLock.writeLock().lock();
-      if (resolved.containsKey(txnID)) {
-        return;
-      }
-
-      resolved.put(txnID, status);
-      recentResolved.add(txnID);
-      if (recentResolved.size() > RESOLVED_TXN_CACHE_SIZE) {
-        Long front = recentResolved.remove();
-        resolved.remove(front);
-      }
-    } finally {
-      readWriteLock.writeLock().unlock();
-    }
-  }
-
-  private Long getResolved(Long txnID) {
-    try {
-      readWriteLock.readLock().lock();
-      return resolved.get(txnID);
-    } finally {
-      readWriteLock.readLock().unlock();
-    }
-  }
-
   private Long getTxnStatus(BackOffer bo, Long txnID, ByteString primary) {
-    Long status = getResolved(txnID);
+    Long status = AbstractLockResolverClient.getResolved(readWriteLock, resolved, txnID);
```

Comment:
A bit strange that we use abstract client to resolve locks...

Reply:
fixed